test/run.sh

```diff
@@ -1,77 +1,49 @@
 #!/bin/bash
 
-dnsmasq_pre() {
-  echo -n $(date +"%b %e %H:%M:%S")
-  echo -n "dnsmasq[123]:"
-}
+# Only run tests on x86_64 target
+if [[ "${1}" != "pihole-FTL-linux-x86_64" ]]; then
+  echo "Skipping tests (${1})!"
+  exit 0
+fi
```

Comment:
Can this check be done only if `CI` is set to `true`? Then it is easier to run the tests locally.